pkg/operator/api/userconfig/apis.go

```diff
@@ -115,7 +114,7 @@ var apiValidation = &cr.StructValidation{
 					{
 						StructField: "SignatureKey",
 						StringValidation: &cr.StringValidation{
-							Default: consts.DefaultTFServingSignatureKey,
+							Required: true,
```

Comment:
Should this be `required: false`? I guess right now it doesn’t matter since this is the only field in the struct, but later if we had more fields could this one be optional?